code/game/machinery/computer/fluffcomputer.dm

```diff
@@ -0,0 +1,141 @@
+/* this entire thing is a non player operatable computer that just gives a access denied message or something like that when you try to use it.
+It does nothing and only serves to look good and for immhulsions. intended for vaults away missions and things of that sort.
+This is not related to actual functional consoles in any way */
+
+/obj/machinery/computer/unusable_communication_relay
+    name = "Communications Relay"
+    desc = "A code locked console that you do not know how to use. Probably doesnt do anything interesting."
+    icon_state = "comm"
+    circuit = "/obj/item/weapon/circuitboard/unusable_communication_relay"
+
+    light_color = LIGHT_COLOR_BLUE
+
+/obj/machinery/computer/unusable_communication_relay/attack_ai(var/mob/user as mob)
+        to_chat(user, "The systems firewall prevents you from accessing it.")
+
+/obj/machinery/computer/unusable_communication_relay/attack_paw(var/mob/user as mob)
+
+    return src.attack_hand(user)
+    return
+
+/obj/machinery/computer/unusable_communication_relay/attack_hand(var/mob/user as mob)
+    say("Access Denied")
+
+
+/obj/machinery/computer/unusable_communication_relay/emag(mob/user as mob)
+    to_chat(user, "You hold the cryptographic sequencer up to the ID scanner and nothing happens. Guess this only works with NT technology ")
```

Comment:
Make this check for whether the machine actually has power and such.

Also, you forgot a period.